build-elements.js

```diff
@@ -0,0 +1,55 @@
+// @ts-nocheck
+
+import fs from 'fs';
+import path from 'path';
+import postcss from 'postcss';
+import tailwind from 'tailwindcss';
+import autoprefixer from 'autoprefixer';
+import cssnano from 'cssnano';
+import atImport from 'postcss-import';
+import presetEnv from 'postcss-preset-env';
+import importSvg from 'postcss-import-svg';
+import { breakpoints as customMedia } from '@fabric-ds/css/colors.js';
+import tailwindConfig from '@fabric-ds/tailwind-config';
```

Comment:
@digitalsadhu What's the difference between contents of`@fabric-ds/tailwind-config` and what's in `@fabric-ds/css/src/utils/tailwind`?

Reply:
A very good question...
* @fabric-ds/css/src/utils/tailwind is not (or should not be) available for direct usage via that path though it should be via @fabric-ds/css/tailwind-config. See package exports [here](https://github.com/fabric-ds/css/blob/01fd80a27461cf0d593911e974853f84963d0681/package.json#L13). 
* `@fabric-ds/tailwind-config` on the other hand.... appears to just be super outdated and not actually maintained anymore. We must have changed that at some point. Good catch, I'll fix

Reply:
See https://github.com/fabric-ds/elements/pull/124/commits/ddf531100ba78b78f5bedc02e20d6ec76db87212